2025-09-13 17:44:25,026 - SourceAnalyzer - DEBUG - logger.py:64 - 파일 읽기 성공: config/parser/sql_keyword.yaml
2025-09-13 17:44:25,055 - SourceAnalyzer - DEBUG - logger.py:64 - YAML 설정 파일 로드 성공: config/parser/sql_keyword.yaml
2025-09-13 17:44:25,055 - SourceAnalyzer - DEBUG - logger.py:64 - 파일 읽기 성공: config/parser/xml_parser_config.yaml
2025-09-13 17:44:25,061 - SourceAnalyzer - DEBUG - logger.py:64 - YAML 설정 파일 로드 성공: config/parser/xml_parser_config.yaml
2025-09-13 17:44:25,062 - SourceAnalyzer - DEBUG - logger.py:64 - 파일 읽기 성공: config/parser/sql_keyword.yaml
2025-09-13 17:44:25,083 - SourceAnalyzer - DEBUG - logger.py:64 - YAML 설정 파일 로드 성공: config/parser/sql_keyword.yaml
2025-09-13 17:44:25,084 - SourceAnalyzer - DEBUG - logger.py:64 - 파일 읽기 성공: config/parser/xml_parser_config.yaml
2025-09-13 17:44:25,090 - SourceAnalyzer - DEBUG - logger.py:64 - YAML 설정 파일 로드 성공: config/parser/xml_parser_config.yaml
2025-09-13 17:44:25,103 - SourceAnalyzer - DEBUG - logger.py:64 - 데이터베이스 연결 성공: projects/sampleSrc/metadata.db
2025-09-13 17:44:25,103 - SourceAnalyzer - DEBUG - logger.py:64 - 쿼리 실행 성공: 
                SELECT COUNT(*) as count 
       ...
2025-09-13 17:44:25,107 - SourceAnalyzer - ERROR - logger.py:128 - FATAL ERROR at D:\Analyzer\CreateMetaDb\parser\xml_parser.py:1255 in _create_inferred_column()
Code: db_utils = DatabaseUtils()
Message: Inferred Column 생성 실패: USERS.DEPT_ID
Exception: TypeError: DatabaseUtils.__init__() missing 1 required positional argument: 'db_path'
Stack Trace:
Traceback (most recent call last):
  File "D:\Analyzer\CreateMetaDb\parser\xml_parser.py", line 1255, in _create_inferred_column
    db_utils = DatabaseUtils()
TypeError: DatabaseUtils.__init__() missing 1 required positional argument: 'db_path'

